include/samplog/LogLevel.h

```diff
@@ -14,8 +14,15 @@ enum samplog_LogLevel
 	INFO = 2,
 	WARNING = 4,
 	ERROR = 8,
+	FATAL = 16,
+	VERBOSE = 32,
 };
 
+inline samplog_LogLevel operator|(samplog_LogLevel a, samplog_LogLevel b)
```

Comment:
Fixed.  I did this originally, but put it within the `samplog` namespace, which actually meant it didn't work correctly so I took it out again.  I didn't think about putting it inside `__cplusplus` but outside the namespace.